lib/samson/hooks.rb

```diff
@@ -110,10 +110,6 @@ def add_decorators
         end
       end
 
-      def add_assets_to_precompile
```

Comment:
why is this no longer required? rails upgrade does it automatically?

Reply:
yeah, it's automatic, either because of the new version or because it was in the wrong position before